src/renderer/gdi/state.cpp

```diff
@@ -300,6 +322,16 @@ GdiEngine::~GdiEngine()
     // Save the font.
     _hfont = hFont.release();

```

Comment:
it is hilarious that we used the smart pointer types for all of the 8 seconds required to ... decapsulate them and store them as raw handles in our class. LOL.

Reply:
Yeah, I mean. I'm sure I had good intentions like "it'll be released properly if one of the interim steps fails and the `RETURN_IF*` macro fires. But it is pretty silly. The class should probably have some smart pointers too...